native/packages/react-native-bpk-component-icon/stories.js

```diff
@@ -18,7 +18,7 @@
 
 import React from 'react';
 import { storiesOf } from '@storybook/react-native';
-import { Platform, View, StyleSheet } from 'react-native';
+import { Platform, View, StyleSheet, Text } from 'react-native';
```

Comment:
Can we use `BpkText` for the story titles?
I realise that's kinda ironic as you made the exact opposite comment to me the other day 😆

Reply:
Point taken, in this one I don't have BpkText as devDependency, in your bit you had it 😄